protocol/avdl/chat1/common.avdl

```diff
@@ -298,6 +298,10 @@
     // server's view of now(), used to calculate ephemeral lifetimes
     @mpackkey("n") @jsonkey("n")
     gregor1.Time now;
+    // clients mark the received time as soon as they pull the message from the
+    // server
+    @mpackkey("rt") @jsonkey("rt")
+    gregor1.Time rtime;
```

Comment:
Ack actually could you make this a nullable thing?

Reply:
I just don't want the server wasting any bandwidth on it

Reply:
yeah